pjrt/pjrt.zig

```diff
@@ -377,7 +410,9 @@ pub const Client = opaque {
         element_type: BufferType,
         layout: MemoryLayout,
         device: *const Device,
-        on_delete_callback: ?*const fn (device_buffer_ptr: ?*anyopaque, ctx: ?*anyopaque) callconv(.C) void = null,
+        on_delete_callback: *const fn (device_buffer_ptr: ?*anyopaque, ctx: ?*anyopaque) callconv(.C) void = &struct {
```

Comment:
why do you put  a callback that does nothing rather than a null ptr ?

Reply:
see https://github.com/zml/zml/pull/180/files#diff-a420f288740d7de49c3cd0b14b3022be15d6d16b4c606172d16b578b726fd5ccL80